experiments/golden-results/StratoX-summary.txt

```diff
@@ -245,7 +220,7 @@ Nkind: N_Pragma
 --
 Occurs: 1 times
 Calling function: Do_Private_Type_Declaration
-Error message: Abstract type declaration unsupported
+Error message: Tagged type declaration unsupported
```

Comment:
Not sure I understand what is going on here.  Can you track down the code that triggers this?